test/runner/lib/cloud/sentry.py

```diff
@@ -0,0 +1,188 @@
+"""Sentry plugin for integration tests."""
+
+from __future__ import absolute_import, print_function
+
+import os
+
+from . import (
+    CloudProvider,
+    CloudEnvironment,
+    CloudEnvironmentConfig,
+)
+
+from ..util import (
+    find_executable,
+    display,
+)
+
+from ..docker_util import (
+    docker_run,
+    docker_rm,
+    docker_inspect,
+    docker_pull,
+    get_docker_container_id,
+)
+
+
+class SentryProvider(CloudProvider):
+    """Sentry plugin.
+
+    Sets up Sentry stub server for tests.
+    """
+
+    DOCKER_SIMULATOR_NAME = 'sentry-stub'
+
+    DOCKER_IMAGE = 'quay.io/ok_go_love/sentry-test-container:9.1.0-clean'
+    """Default image to run Storage stub from.
+
+    It's source source itself resides at:
+    https://github.com/okgolove/docker-test-container
```

Comment:
I guess this is https://github.com/okgolove/sentry-test-container now? Because the URL above gives a 404.

Reply:
Nice catch.